src/material/icon/icon.scss

```diff
@@ -59,6 +59,26 @@ mat-icon {
 
     .mat-icon-button .mat-icon {
       margin: auto;
+      overflow: visible;
     }
   }
 }
+
+// Makes button icon overflow visible to fix b/250063405
+.mat-mdc-icon-button .mat-icon,
+.mdc-button .mat-icon,
+.mat-mdc-fab-base .mat-icon,
+.mat-toolbar .mat-icon,
+.mat-mdc-tooltip-trigger .mat-icon,
+.mat-mdc-tab .mat-icon,
+.mat-mdc-form-field .mat-icon,
+.mat-grid-tile .mat-icon,
+.mat-mdc-list-item .mat-icon,
+.mat-mdc-list-option .mat-icon,
+.mat-icon:has(div[style*='letter-spacing:']):has(div[style*='word-spacing:']) {
```

Comment:
I think that we should drop the `.mat-icon:has(div[style*='letter-spacing:']):has(div[style*='word-spacing:']` style. If there's some internal app setting these as inline styles, we should fix the app rather than our styles.